src/index.html

```diff
@@ -4,9 +4,371 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet"> 
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page-content">
+    <header class="header page-content__header">
+      <div class="header__logo-wrapper">
+        <img 
+          src="./images/logo.png" 
+          alt="Moyo logo" 
+          class="header__logo"
+        >
+      </div>
+      <nav>
+        <ul class="header__menu">
+          <li class="header__menu-item">
+            <a 
+              href="#apple" 
+              class="header__menu-link is-active"
+            >Apple</a>
+          </li>
+
+          <li class="header__menu-item">
+            <a href="#Samsung" class="header__menu-link">Samsung</a>
+          </li>
+
+          <li class="header__menu-item">
+            <a href="#smartphones" class="header__menu-link">Smartphones</a>
+          </li>
+
+          <li class="header__menu-item" >
+            <a 
+              href="#laptopsAndComputers" 
+              class="header__menu-link"
+              data-qa="nav-hover"
+            >Laptops & Computers</a>
+          </li>
+
+          <li class="header__menu-item">
+            <a href="#gadgest" class="header__menu-link">Gadgets</a>
+          </li>
+
+          <li class="header__menu-item">
+            <a href="#tablets" class="header__menu-link">Tablets</a>
+          </li>
+
+          <li class="header__menu-item">
+            <a href="#photo" class="header__menu-link">Photo</a>
+          </li>
+
+          <li class="header__menu-item">
+            <a 
+              href="#video" 
+              class="header__menu-link header__menu-link--last"
+            >Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="page-main">
+      <section class="cards-section page-main__cards-section">
+        <div class="product-card cards-section__product-card" data-qa="card">
+          <img
+            src="./images/imac.jpeg" 
+            class="product-card__image" 
+            alt="APPLE A1419 iMac 27 Retina 5K"
+          >
+          <h2 class="product-card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="product-card__code">
+            Product code: 195434
+          </p>
+          <div class="product-card__reviews-wrapper">
```

Comment:
The same thing I mentioned earlier - remember to scan your code for similar mistakes after each review
```suggestion
          <div class="product-card__reviews">
```